libs/portal-integration-angular/src/lib/core/components/search-config/search-config.component.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core'
+import { SearchConfig } from '../../../model/search-config'
+import { FormControl, FormGroup } from '@angular/forms'
+
+@Component({
+  selector: 'ocx-search-config',
+  templateUrl: './search-config.component.html',
+  styleUrls: ['./search-config.component.scss'],
+})
+export class SearchConfigComponent implements OnInit {
+  @Input()
+  searchConfigs: SearchConfig[] | [] | undefined
+
+  @Input() defaultTranslationKey = 'OCX_SEARCH_HEADER.OCX_SEARCH_CONFIG.DROPDOWN_DEFAULT'
```

Comment:
Better placeholderKey or something like that. Default implies that it can not be changed

Reply:
done